yo_trombinoscope_member/__manifest__.py

```diff
@@ -17,13 +17,5 @@
         'views/snippets/s_trombinoscope_member.xml',
         'views/snippets/snippets.xml',
     ],
-    "assets": {
```

Comment:
Asset still use,
See changelog
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0